Wikipedia/Code/WMFFeedContentFetcher.m

```diff
@@ -90,6 +90,17 @@ - (void)fetchFeedContentForURL:(NSURL *)siteURL date:(NSDate *)date force:(BOOL)
                                           userInfo:nil]);
 
             } else {
+                NSHTTPURLResponse *response = ((NSHTTPURLResponse *)[operation response]);
+                NSDictionary *headers = [response allHeaderFields];
+                NSString *cacheControllHeader = headers[@"Cache-Control"];
+
+                NSRange range1 = [cacheControllHeader rangeOfString:@"max-age="];
+                NSRange range2 = NSMakeRange(cacheControllHeader.length, 1);
+                NSRange range = NSMakeRange(range1.location + range1.length, range2.location - range1.location - range1.length);
+                NSInteger maxAge = [[cacheControllHeader substringWithRange:range] intValue];
```

Comment:
This will fail if the order changes - for example if the header becomes `max-age=10, s-maxage=100`, should be updated to handle this case - maybe a regex?

Reply:
I swapped it for `(?<=max-age=)\\d{2}` but is there a way to match all the digits after the string, not just 2?